src/vma/dev/cq_mgr.cpp

```diff
@@ -127,6 +127,7 @@ inline void cq_mgr::compensate_qp_poll_failed()
 		if (likely(m_rx_pool.size() || request_more_buffers())) {
 			do {
 				mem_buf_desc_t *buff_new = m_rx_pool.get_and_pop_front();
+				buff_new->reset_ref_count();
```

Comment:
@liranoz12, __request_more_buffers()__ is responsible to provide more buffers from global pool to r_rx_pool which will be used for processing. The point here was to have initialization of ref counter in place where we take buffer from m_rx_pool to make sure new buffer taken from the pool is in correct shape. Another case here request will happen if there some free buffers in m_rx_pool so getting it in unknown state is not safe ether. Actually I had also tried this approach
but got time degradation, mostly I think because old buffers are in cache but processing of new will flush it. But it is just guess.